src/smt/solver_engine.h

```diff
@@ -1010,6 +1010,17 @@ class CVC5_EXPORT SolverEngine
   /** Vector version of above. */
   void ensureWellFormedTerms(const std::vector<Node>& ns,
                              const std::string& src) const;
+  /**
+   * Convert preprocessed assertions to unsat core. This converts a
```

Comment:
I think this documentation is not very clear. My understanding is that this method *creates* an unsat core by *connecting* the preprocessed assertions to the input assertions via the proof of processing. Changing the name of the method and the explanation could clarify this.